pages/styles/login.css

```diff
@@ -0,0 +1,33 @@
+.auth-form {
+  margin-top: 5rem;
+  height: 100%;
+}
+
+.auth-form form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 2px solid var(--color-primary);
+  border-radius: 0.5rem;
+  padding: 1rem;
+}
+
+.auth-form form .inp-unit {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+}
+
+.login-btn {
+  padding: 0.5rem !important;
+}
+
+.text-center {
+  text-align: center;
+}
+
+input {
```

Comment:
Better to use dedicated class names for 2 reasons :- 1. Consistency. 2. To avoid conflicting as project gets bigger.

Reply:
Thanks, I've fixed it.